ext/GradedArraysTensorAlgebraExt/GradedArraysTensorAlgebraExt.jl

```diff
@@ -23,16 +23,17 @@ struct SectorFusion <: FusionStyle end
 
 TensorAlgebra.FusionStyle(::Type{<:GradedArray}) = SectorFusion()
 
-# TODO consider heterogeneous sectors?
+# TBD consider heterogeneous sectors?
 TensorAlgebra.trivial_axis(t::Tuple{Vararg{AbstractGradedUnitRange}}) = trivial(first(t))
 
 function matricize_axes(
   blocked_axes::BlockedTuple{2,<:Any,<:Tuple{Vararg{AbstractUnitRange}}}
 )
+  @assert !isempty(blocked_axes)
+  default_axis = trivial_axis(Tuple(blocked_axes))
   codomain_axes, domain_axes = blocks(blocked_axes)
-  @assert !(isempty(codomain_axes) && isempty(domain_axes))
-  row_axis = unmerged_tensor_product(trivial_axes(domain_axes), codomain_axes...)
-  unflipped_col_axis = unmerged_tensor_product(trivial_axes(codomain_axes), domain_axes...)
+  row_axis = unmerged_tensor_product(default_axis, codomain_axes...)
+  unflipped_col_axis = unmerged_tensor_product(default_axis, domain_axes...)
```

Comment:
```suggestion
  codomain_axis = unmerged_tensor_product(default_axis, codomain_axes...)
  unflipped_domain_axis = unmerged_tensor_product(default_axis, domain_axes...)
```
and elsewhere. I would prefer to stick with the terminology "codomain" and "domain" for consistency.